eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/fixtures/MinimalNodeFixture.scala

```diff
@@ -277,7 +278,7 @@ object MinimalNodeFixture extends Assertions {
     val invoice = sender.expectMsgType[Bolt11Invoice]
 
     val routeParams = node1.nodeParams.routerConf.pathFindingExperimentConf.experiments.values.head.getDefaultRouteParams
-    sender.send(node1.paymentInitiator, PaymentInitiator.SendPaymentToNode(amount, invoice, maxAttempts = 1, routeParams = routeParams, assistedRoutes = hints, blockUntilComplete = true))
+    sender.send(node1.paymentInitiator, PaymentInitiator.SendPaymentToNode(amount, invoice, maxAttempts = 1, routeParams = routeParams, extraEdges = hints.flatMap(Bolt11Invoice.toExtraEdges(_, node2.nodeParams.nodeId)).map(GraphEdge(_)), blockUntilComplete = true))
```

Comment:
I think it would make more sense if this function directly took a `Seq[GraphEdge]`, since the way we're building those hints from a private channel in `ZeroConfAliasIntegrationSpec` would make it easy to directly build a `GraphEdge` there.